ios/brave-ios/Sources/AIChat/Resources/tr.lproj/BraveLeo.strings

```diff
@@ -0,0 +1,291 @@
+/* The title for button that submits feedback */
+"aichat.addFeedbackActionTitle" = "Geri Bildirim Ekle";
+
+/* The title for the settings to show search suggestions from Leo in url-address bar */
+"aichat.advancedSettingsAutocompleteTitle" = "Adres çubuğunda otomatik tamamlamayı göster";
+
+/* The title for the settings to change default model for conversations */
+"aichat.advancedSettingsDefaultModelTitle" = "Yeni konuşmalar için varsayılan model";
+
+/* The title for the header for adjusting leo ai settings */
+"aichat.advancedSettingsHeaderTitle" = "Leo, gücünü yapay zekâdan alan bir akıllı asistandır. Doğrudan tarayıcınıza dâhil edilmiştir.";
+
+/* The subtitle for the button which links purchase to Brave Account */
+"aichat.advancedSettingsLinkPurchaseActionSubTitle" = "Leo'yu diğer cihazlarda kullanmak için Appstore satın alımınızı Brave hesabınıza bağlayın.";
+
+/* The title for the button which links purchase to Brave Account */
+"aichat.advancedSettingsLinkPurchaseActionTitle" = "Satın alımı Brave hesabınıza bağlayın";
+
+/* The title for the label showing the date which subscription expires */
+"aichat.advancedSettingsSubscriptionExpiresTitle" = "Sona erme tarihi";
+
+/* The title for the header for subscription details */
+"aichat.advancedSettingsSubscriptionHeaderTitle" = "Abonelik";
+
+/* The title for the label showing status if the subscription */
+"aichat.advancedSettingsSubscriptionStatusTitle" = "Durum";
+
+/* The subtitle for the error showing there is an erro from Appstore */
+"aichat.appStoreErrorSubTitle" = "Appstore'a bağlanılamadı, lütfen daha sonra tekrar deneyin.";
+
+/* The title for the error showing there is an erro from Appstore */
+"aichat.appStoreErrorTitle" = "App Store Hatası";
+
+/* The title of the search bar suggestion when prompting to Ask Leo */
+"aichat.askLeoSearchSuggestionTitle" = "Leo'ya Sorun";
+
+/* The subtitle for intro view which triggers just chat */
+"aichat.chatIntroJustTalkSubTitle" = "Bana istediğinizi sorun! İstediğiniz konu hakkında konuşabiliriz. Size daha iyi yanıt verebilmek için sürekli öğreniyor ve kendimi geliştiriyorum.";
+
+/* The title for intro view which triggers just chat */
+"aichat.chatIntroJustTalkTitle" = "Sadece sohbet etmek mi istiyorsunuz?";
+
+/* The subtitle for intro view */
+"aichat.chatIntroSubTitle" = "Doğrudan Brave'in içine inşa edilen, YZ destekli akıllı bir asistanım.";
```

Comment:
"Doğrudan Brave'in içine inşa edilen, Yapay Zeka destekli akıllı bir asistanım.";